concept/type_/owns.rs

```diff
@@ -48,6 +54,25 @@ impl<'a> Owns<'a> {
         }
     }
 
+    // TODO: Should it be 'this or just 'tm on type_manager?
```

Comment:
Lifetimes question.

Reply:
I think it'll be OK if we return 'this for the getters